flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterConfigurationParserFactory.java

```diff
@@ -45,12 +45,29 @@
 
 	static final JobID DEFAULT_JOB_ID = new JobID(0, 0);
 
-	private static final Option JOB_CLASS_NAME_OPTION = Option.builder("j")
+	@Deprecated
+	private static final Option JOB_CLASS_NAME_OPTION = Option.builder("jc")
 		.longOpt("job-classname")
 		.required(false)
 		.hasArg(true)
 		.argName("job class name")
-		.desc("Class name of the job to run.")
+		.desc("Class name of the job to run. (deprecated; use job-entrypoint instead.)")
+		.build();
+
+	private static final Option JOB_ENTRYPOINT_NAME_OPTION = Option.builder("j")
+		.longOpt("job-entrypoint")
+		.required(false)
+		.hasArg(true)
+		.argName("job entrypoint name")
+		.desc("Entrypoint name of the job to run.")
+		.build();
+
+	private static final Option JOB_PYTHON_ARTIFACTS_OPTION = Option.builder("ja")
```

Comment:
Why does this need to be handled by the `StandaloenJobClusterConfiguration` and cannot be parsed by the `PythonDriver` program?